internal/repositories/repositories.go

```diff
@@ -62,13 +60,13 @@ func (r *Repositories) getCacheDir() (string, error) {
 		return "", err
 	}
 
-	cacheDir := filepath.Join(giltDir, "cache")
-	if _, err := r.appFs.Stat(cacheDir); os.IsNotExist(err) {
-		if err := r.appFs.Mkdir(cacheDir, 0o700); err != nil {
-			return "", err
+	cacheDir := r.appFs.Join(giltDir, "cache")
+	if err := r.appFs.MkdirAll(cacheDir, 0o700); err != nil {
+		if i, e := r.appFs.Stat(cacheDir); e == nil && i.IsDir() {
+			return cacheDir, nil
 		}
+		return "", err
 	}
-
 	return cacheDir, nil
```

Comment:
This code right here is what should fill you with skepticism.  avfs has some, shall we say, eccentricities around `ENOENT` errors from the various backends.  And they have hinted that they do not consider OSX a supported platform at all.

It does seem to work, and so long as file tests are evaluated in the affirmative, it seems to work well.  But it should, at least a little, give you pause.